tests/managers/vs/vs_tests.py

```diff
@@ -830,3 +830,13 @@ def test_capture_additional_disks(self):
                                 'createArchiveTransaction',
                                 args=args,
                                 identifier=1)
+
+    def test_usage_vs(self):
+        result = self.vs.get_summary_data_usage('100',
+                                                start_date='2019-3-4',
+                                                end_date='2019-4-2',
+                                                valid_type='CPU0',
+                                                summary_period=300)
+
+        self.assertEqual(result, [{'counter': 1.44, 'dateTime': '2019-03-04T00:00:00-06:00', 'type': 'cpu0'},
```

Comment:
unless the manager is making changes to the returned data, you can just `self.assert_called_with()` to make sure your command is working.

Reply:
Done.